### PR TITLE
Folder Added:HTML Assignment

### PR DESCRIPTION
**Pull request by pritikekan**

### CODE REVIEW

**Review by anuja1603 — APPROVED** [review]

**Review by pooja-nagare — APPROVED** [review]

**Review by shivani-pawbake — APPROVED** [review]

**Review by Amit727619 — APPROVED** [review]

**Review by jyoti-ing — APPROVED** [review]

File: priti/HTML/Assign. 2/Que1.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE>
+
+<html>
+    <head>
+        <title> Html Heading</title>
+    </head>
+    <body style="background-color:powderblue">
+
```
Comment (JafarPathan) [inline priti/HTML/Assign. 2/Que1.html]: removed extra lines

File: priti/HTML/Assign. 2/Que1.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <h1 style="text-align:center; color:red">Google</h1>
+        <p><i>This article is about the company</i></p>
+        <h1>From the garage to the Googleplex</h1>
+        <p><big>The Google story begins in 1995 at <b> Standford University.</b></big></p>
+        <p><big><b><i>Larry Page </i></b>was considering Stanford for grad school and </big><b><i><big>Sergey Brin,</i></b> a student there, was assigned to show him around.</big></p>
+        <p>By some accounts, they disagreed about nearly everything during that first meeting, but by the following
+            year they struck a .</p>
+            <p> Working from their dorm rooms, they built a search engine that used links to determine the importance of<br>
+                individual pages on the <span style = "color:blue">World Wide Web. </span></p>
+                <p>  They called this search engine <cite>Backrub.</cite></p>
```
Comment (JafarPathan) [inline priti/HTML/Assign. 2/Que1.html]: check indentation

File: priti/HTML/Assign. 2/Que1.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        
+        
+            
+        
+        
+
+
+
+
+
```
Comment (JafarPathan) [inline priti/HTML/Assign. 2/Que1.html]: remove extra line

File: priti/HTML/Assign.1/Assign1.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <title> Assignment 1</title>
+    </head>
+
```
Comment (JafarPathan) [inline priti/HTML/Assign.1/Assign1.html]: remove line

File: priti/HTML/Assign.1/Assign1.html [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+        <h1><b> HTML</b></h1>
+
+
+        <h4>Introduction</h4>
+        <p> HTML is a standard markup language for web pages.</p>
+
+        
+        </body>
+    
+</html>
```
Comment (JafarPathan) [inline priti/HTML/Assign.1/Assign1.html]: add extra line at end

File: priti/HTML/Assign.1/Assign4.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <title> Assignment 4</title>
+    </head>
+
+        <body>
+        <h1><b> PRITI</b></h1>
+        <p><b>MY name is priti. I am 22yrs old from nashik</p>
```
Comment (JafarPathan) [inline priti/HTML/Assign.1/Assign4.html]: check tags

File: priti/HTML/Assign.3/pritiA3_2contact.html [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+      us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+      us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact
+      us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+      us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact
+      us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+      us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact
+      us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.
+      Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+      us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact
+      us.Contact us.Contact us.Contact us.Contact us.
```
Comment (JafarPathan) [inline priti/HTML/Assign.3/pritiA3_2contact.html]: .......?

**Review by JafarPathan — COMMENTED** [review]
Check indentation in all file use 4 spaces for indentation
add extra line at the end of files 
try to implement only one type of css

**Review by jyoti-ing — CHANGES_REQUESTED** [review]
remove extra lines
ques1.html

**Review by JafarPathan — APPROVED** [review]

**Pull request merged**